docs/_user_guide/python.md

```diff
@@ -0,0 +1,114 @@
+# Running Python Interactively
+
+On the Yens, Python is available via the `python3` command, a version different from the system Python can be chosen via the `module` command.
+
+See all currently installed versions with:
+
+```title="Terminal Input"
+module avail python/
+```
+
+
+You will see the current Python versions listed:
+
+```{.yaml .no-copy title="Terminal Output"}
+------------------------------------------------------------ Global Aliases ------------------------------------------------------------
+
+----------------------------------------------------------- /software/modules/Core -----------------------------------------------------
+   intel-python/2019.4    intel-python3/2019.4    python/2.7.18    python/3.10.5 (D)    python/3.10.11    python/3.11.3
+
+  Where:
+   D:  Default Module
+```
+
+!!! tip
+    If you require access to a newer Python version that is not currently available on the system, please don't hesitate to [contact DARC](mailto:gsb_darcresearch@stanford.edu) to request its installation.
+
+
+To use the system Python software on the Yens, simply type `python3`.
+
+```title="Terminal Input"
+python3
+```
+
+This Python can be found at `/usr/bin/python3`
+
+You should see:
+
+```{.yaml .no-copy title="Terminal Output"} 
+Python 3.10.12 (main, Sep 11 2024, 15:47:36) [GCC 11.4.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>> 
+```
+
+This will load the system Python version (Python-3.10.12) 
+You can also load a specific version of Python with:
+
+```title="Terminal Input"   
+ml python/3.10.11
+python3
+```
+!!! Note
+    ml is an alias for the `module load` command.
+
+Start the Python interactive console by typing `python3`.
+You should see:
+    
+```{.yaml .no-copy title="Terminal Output"} 
+Python 3.10.11 (main, May  6 2023, 14:44:19) [GCC 11.3.0] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>>
+
+```
+
+### Managing Python Software Libraries and Versions
```

Comment:
that's not right

Reply:
What is not right?

Reply:
Hey! I mean the whole page. We don't recommend module for python at all

Reply:
and you copied the R section but it does not apply to python libraries